build.gradle

```
@@ -101,6 +101,7 @@ dependencies {
     compile group: 'org.terasology', name: 'CrashReporter', version: '1.2.+'
     compile group: 'com.github.rjeschke', name: 'txtmark', version: '0.11'
 
+    compile 'com.google.code.gson:gson:2.8.5'
```

Please follow the same pattern as all the other dependencies.
```suggestion
    compile group: 'com.google.code.gson', name: 'gson', version: '2.8.5' 
```